src/main/java/com/incognia/api/IncogniaAPI.java

```diff
@@ -72,7 +92,8 @@ public class IncogniaAPI {
    * @return the singleton instance
    */
   public static IncogniaAPI init(String clientId, String clientSecret) {
-    INSTANCE.compareAndSet(null, new IncogniaAPI(clientId, clientSecret));
+    INSTANCE.compareAndSet(
```

Comment:
nit: this should call the other `init` function with `CustomOptions.builder().build()`

Reply:
`return init(clientId, clientSecret, CustomOptions.builder().build()`

Reply:
done!